state/factory/workingset.go

```diff
@@ -32,105 +27,74 @@ var (
 		},
 		[]string{"type"},
 	)
-	dbBatchSizelMtc = prometheus.NewGaugeVec(
-		prometheus.GaugeOpts{
-			Name: "iotex_db_batch_size",
-			Help: "DB batch size",
-		},
-		[]string{},
-	)
 )
 
 func init() {
 	prometheus.MustRegister(stateDBMtc)
-	prometheus.MustRegister(dbBatchSizelMtc)
 }
 
 type (
```

Comment:
would suggest we avoid keep changing these interface radically ...

understand that the workingset implementation can be refactored using these get/put/delete/commitFunc(), and NewWorkingSet() replaced by workingSetCreator interface

but the whole team need to keep chasing and understand the new change ...